include/graphics.c

```diff
@@ -117,6 +145,56 @@ void InitStdText() {
     SDL_BlitSurface(copyright, NULL, bgSurface, &copyrightPos);
     SDL_FreeSurface(copyright);
 
+    SDL_Rect controls1Pos = {windowWidth-170, 310, 100, 60};
+    SDL_Surface *controls1 = TTF_RenderText_Blended_Wrapped(ultraSmallFont, "D-Pad or arrow keys to move", textColor, 120);
+    SDL_BlitSurface(controls1, NULL, bgSurface, &controls1Pos);
+    SDL_FreeSurface(controls1);
+
+    SDL_Rect controls2Pos = {5, 320, 100, 60};
+    SDL_Surface *controls2 = TTF_RenderText_Blended_Wrapped(ultraSmallFont, "A or Enter to Confirm selection", textColor, 120);
+    SDL_BlitSurface(controls2, NULL, bgSurface, &controls2Pos);
+    SDL_FreeSurface(controls2);
+
+}
+
+// Calculates a number that can be used for x or y, used to calculate letter positions
+// Takes a field line width and a character width, in any order
+int calculatePos(int num, int width1, int width2) {
+    // If the number is out of bounds, return maximum/minimum possible
+    if (num > 2) {
+        return (2*width1)+(2*width2);
```

Comment:
maybe we should switch to asserts here? 
Ofc, asserts on the xbox suck, because the system just hangs. But on the other hand, with the current code, bugs might go unnoticed.